lib/nexmo_markdown_renderer/services/code_snippet_renderer/curl.rb

```diff
@@ -22,6 +22,10 @@ def self.create_instructions(filename)
         def self.add_instructions(filename)
           ::I18n.t('services.code_snippet_renderer.add_instructions_to_file', file: filename)
         end
+
+        def self.post_process(code)
```

Comment:
Is `#post_process` solely for managing the single quote marks? If so, can I suggest a method that strips the single quotes, and then that method gets invoked here? Otherwise, the name is too general for what the method does.

```ruby
def self.post_process(code)
  self.strip_single_quotes(code)
end

def self.strip_single_quotes(code)
  code.gsub(/"'(\$\w+)'"/, '"\1"')
end
```

Reply:
It's a general post processing method. For `curl` it strips single quotes, but for other renderers it could perform other actions